docs/rules/comma-style.md

```diff
@@ -1,4 +1,4 @@
-# Comma style (comma-style)
+# Enforce One True Comma Style (comma-style)
```

Comment:
Oh, interesting. I wonder if this is the intended name (I see that you're making it consistent with the description in the main readme file), @eslint/eslint-team?